src/nft/components/explore/Table.tsx

```diff
@@ -0,0 +1,112 @@
+import clsx from 'clsx'
+import { Column, IdType, Row, useSortBy, useTable } from 'react-table'
+
+import { Box } from '../../components/Box'
+import { CollectionTableColumn } from '../../types'
+import { ArrowRightIcon } from '../icons'
+import { ColumnHeaders } from './CollectionTable'
+import * as styles from './Explore.css'
+
+interface TableProps<D extends Record<string, unknown>> {
+  columns: Column<CollectionTableColumn>[]
+  data: CollectionTableColumn[]
+  hiddenColumns: IdType<D>[]
+  classNames?: {
+    td: string
+  }
+  getTrProps: (row: Row<CollectionTableColumn>) => Record<string, unknown>
+}
+
+export function Table<D extends Record<string, unknown>>({
+  columns,
+  data,
+  hiddenColumns,
+  classNames,
+  getTrProps,
+  ...props
+}: TableProps<D>) {
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } = useTable(
+    {
+      columns,
+      data,
+      initialState: {
+        sortBy: [
+          {
+            desc: true,
+            id: ColumnHeaders.Volume,
+          },
+        ],
+      },
+      getTrProps,
+      ...props,
+    },
+    useSortBy
+  )
+
+  /*
+  useEffect(() => {
+    if (hiddenColumns && isMobile) {
+      setHiddenColumns(hiddenColumns)
+    } else {
+      setHiddenColumns([])
+    }
+  }, [isMobile, hiddenColumns, setHiddenColumns])
+  */
+
+  return (
+    <table {...getTableProps()} className={styles.table}>
+      <thead className={styles.thead}>
+        {headerGroups.map((headerGroup) => (
+          <tr {...headerGroup.getHeaderGroupProps()} key={headerGroup.id}>
+            {headerGroup.headers.map((column, index) => {
+              return (
+                <th
+                  className={styles.th}
+                  {...column.getHeaderProps(column.getSortByToggleProps())}
+                  style={{
+                    textAlign: index === 0 ? 'left' : 'right',
+                    paddingLeft: index === 0 ? '52px' : 0,
+                  }}
+                  key={index}
+                >
+                  <Box as="span" color="blue400" position="relative">
```

Comment:
is this color the same in dark and light mode? if this changes I believe it would be `backgroundAction`